README.md

```diff
@@ -15,10 +15,10 @@ To start ILP Kit, run the command below, but with your own values instead of the
 placeholders.
 
 ```
```

Comment:
aah, ok, I had ignored the `\`, that's why my env vars weren't sticking. nevermind!